src/v2/Apps/Auction/Components/AuctionDetails/AuctionDetails.tsx

```diff
@@ -65,6 +72,14 @@ const AuctionDetails: React.FC<AuctionDetailsProps> = ({ sale, me }) => {
         </Flex>
       </Flex>
 
+      <Flex alignItems="center" justifyContent="space-between">
+        <Text variant="md" pr={2}>
+          {!!sale.cascadingEndTimeInterval &&
+            !sale.endedAt &&
+            sale.cascadingEndTime?.intervalLabel}
+        </Text>
+      </Flex>
```

Comment:
Same here, don't think the flex container is necessary since all we're doing is rendering out one text label. Can we omit it completely? And if not, we'll want to 
- store the label in a variable so that its a bit more readable for the future
```
const cascadingEndTimeIntervalLabel = !!sale.cascadingEndTimeInterval && !sale.endedAt && sale.cascadingEndTime?.intervalLabel
```
and then later, use it as the outermost bit to guard against empty divs:

```tsx
{cascadingEndTimeIntervalLabel && 
  <Text variant="md" pr={2}>{cascadingEndTimeIntervalLabel}</Text?
}
```

Reply:
that works! Good call!